modules/libcom/RTEMS/rtems_netconfig.c

```diff
@@ -15,8 +15,27 @@
  */
 #include <stdio.h>
 #include <bsp.h>
+
+#ifndef VERSION_INT
+#  define VERSION_INT(V,R,M,P) ( ((V)<<24) | ((R)<<16) | ((M)<<8) | (P))
```

Comment:
`VERSION_INT()` is available in epicsVersion.h, please include that instead of redefining it here.